lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -2307,6 +2316,29 @@ def visit_create_index(self, create, **kw):
 
         return text
 
+    def visit_unique_constraint(self, constraint, **kw):
+        if len(constraint) == 0:
+            return ""
+        nulls_not_distinct_option = ""
+        nulls_not_distinct = constraint.dialect_options["postgresql"][
```

Comment:
**mike bayer** (zzzeek) wrote:

does this return None if not present?

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4632

Reply:
Yes. And returns `True` or `False` if presented ones